docs/upgrading-flatcar.md

```diff
@@ -13,14 +13,83 @@ Occasionally, changes are made to Flatcar Butane configs (those that specify `va
 1. TOC
 {:toc}
 
-{% comment %}
-
 ## From Version 1.0.0 to Version 1.1.0
 
 There are no breaking changes between versions 1.0.0 and 1.1.0 of the `flatcar` configuration specification. Any valid 1.0.0 configuration can be updated to a 1.1.0 configuration by changing the version string in the config.
 
 The following is a list of notable new features.
 
+### LUKS discard
+
+The `luks` section gained a new `discard` field. If specified and true, the LUKS volume will issue discard commands to the underlying block device when blocks are freed. This improves performance and device longevity on SSDs and space utilization on thinly provisioned SAN devices, but leaks information about which disk blocks contain data.
+
+<!-- butane-config -->
+```yaml
+variant: flatcar
+version: 1.1.0
+storage:
+  luks:
+    - name: luks-static
+      device: /dev/sdb
+      discard: true
+      key_file:
+        inline: REPLACE-THIS-WITH-YOUR-KEY-MATERIAL
+```
+
+### LUKS open options
+
+The `storage.luks` section gained a new `open_options` field. It is a list of options Ignition should pass to `cryptsetup luksOpen` when unlocking the volume. Ignition also passes `--persistent`, so any options that support persistence will be saved to the volume and automatically used for future unlocks. Any options that do not support persistence will only be applied to Ignition's initial unlock of the volume.
```

Comment:
Nit: for Flatcar there's only one `luks` section so let's s/storage.luks/luks/.

Reply:
Hmm, so I did make a change to indicate its just 'luks' and not `storage.luks` simmilarly to the LUKS discard section.

If thats wrong lmk.